roadmap/implementers-guide/src/node/disputes/README.md

```diff
@@ -2,5 +2,48 @@
 
 Handles everything around disputes, ranging from persisting and retrieving
 votes to notifications of other peers and subsystems about the resolution
-as well as black listing invalid blocks and creating a transaction to
-persist the dispute resolution.
\ No newline at end of file
+as well as passing on information to the runtime.
+
+
+`VotesDB` is soley responsible for data persistance in a manner,
+that allows easy retrieval and evaluates the state of particular
+disputes and generates `StoreVoteResult`s.
+
+`Participation` uses runtime `Events` to facility information
+distribution via the transaction gossip.
+As such it queries all `DisputeEvents` on view change
+and extracts `Vote`s on which it operates on.
+
+If a new dispute is detected, it does the `PoV` retrieval and verification based on the runtime verification code and notifies
```

Comment:
I guess this should also mention that this issues new votes based on active disputes, right?